packages/web/app/components/Charting/ChartInstanceInfoSection.jsx

```diff
@@ -89,6 +89,13 @@ export const ChartInstanceInfoSection = ({
     subtypeDataElement,
   } = coreComplexDataElements;
 
+  const editedObject = {
+    ...(instanceNameDataElement?.id && { [instanceNameDataElement.id]: chartInstanceName }),
+    ...(dateDataElement?.id && { [dateDataElement.id]: chartDate }),
+    ...(isTypeVisible && typeDataElement?.id && { [typeDataElement.id]: chartType }),
+    ...(isSubtypeVisible && subtypeDataElement?.id && { [subtypeDataElement.id]: chartSubtype }),
+  };
```

Comment:
```suggestion
  const editedObject = {};
  if (instanceNameDataElement?.id) {
    editedObject [instanceNameDataElement.id] = chartInstanceName;
  }
  
  if (dateDataElement?.id) {
    editedObject[dateDataElement.id] = chartDate;
  }
  
  if (isTypeVisible && typeDataElement?.id) {
    editedObject[typeDataElement.id] = chartType;
  }
  
  if (isSubtypeVisible && subtypeDataElement?.id) {
    editedObject[subtypeDataElement.id] = chartSubtype;
  }
```

Totally optional and admittedly less concise, but I think in these scenarios I prefer to differentiate the checks and the assignment as it reads a bit more clearly to me. :+1: